src/conf.c

```diff
@@ -480,6 +483,7 @@ static void initialize_config_defaults (struct config_s *conf)
          * Make sure the HTML error pages array is NULL to begin with.
          * (FIXME: Should have a better API for all this)
          */
+        conf->basicauth_realm = safestrdup (PACKAGE_NAME);
```

Comment:
last nitpick: this should be moved a line down because the comment above refers to the variable errorpages

Reply:
Ahh, i had not recognized that this comment is about the very next line only.
Got it.